dev/wlp-mavenRepoTasks/src/com/ibm/ws/wlp/mavenFeatures/LibertyFeaturesToMavenRepo.java

```diff
@@ -252,6 +254,12 @@ private void generatePom(LibertyFeature feature, List<MavenCoordinates> featureC
 		model.setPackaging(type.getType());
 		setLicense(model, coordinates.getVersion(), true, feature.isRestrictedLicense(), Constants.WEBSPHERE_LIBERTY_FEATURES_GROUP_ID.equals(coordinates.getGroupId()));
 		
+		Developer dev = new Developer();
```

Comment:
Declare the `dev` object inside the `setScmDevUrl` method, since the dev object doesn't need to be used here.